app/models/notification_blueprint.rb

```diff
@@ -18,6 +18,14 @@ class NotificationBlueprint < ActiveRecord::Base
   validates :expires_in, :numericality => {:greater_than => 0}
   before_validation :set_default_expiry
 
+  def mass_update_expiry
```

Comment:
also, map will create a rails ar object, the approach is mass update on the db level.